_data/whats-new.yml

```diff
@@ -4,8 +4,179 @@ description: |
+    for the Magento 2.2.10 release.
+  versions: 2.2.10
+  type: Major update
+  date: October 14, 2019
+  link: https://github.com/magento/devdocs/pull/5640
+- description: Clarified the use of a static content deployment command for [troubleshooting
+    theme deployment](https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/themes/theme-create.html#trouble).
+  versions: 2.2.x, 2.3.x
+  type: Technical changes
+  date: October 14, 2019
+  link: https://github.com/magento/devdocs/pull/5565
+- description: Added [Backward incompatible changes reference](https://devdocs.magento.com/guides/v2.3/release-notes/backward-incompatible-changes/reference.html)
+    for the Magento 2.3.3 release.
+  versions: 2.3.3
+  type: Major update
+  date: October 14, 2019
+  link: https://github.com/magento/devdocs/pull/5686
+- description: Added a warning to ensure that the [CRYPT_KEY](https://devdocs.magento.com/guides/v2.3/cloud/env/variables-deploy.html#crypt_key)  deploy
+    variable for Magento Commerce Cloud environments is set in the Project Web UI
+    to avoid exposing sensitive data.
+  versions: 2.x
+  type: Technical changes
+  date: October 11, 2019
+  link: https://github.com/magento/devdocs/pull/5663
+- description: Added documentation for the [Folder tree widget](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget-folder-tree.html)
+    to the _Javascript Developer Guide_.
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: October 11, 2019
+  link: https://github.com/magento/devdocs/pull/5503
+- description: Added the "Execute data-mage-init and x-magento-init in dynamic content"
+    section to the [Calling and initializing JavaScript](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/javascript/js_init.html)
+    topic in the _Javascript Developer Guide_.
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: October 11, 2019
+  link: https://github.com/magento/devdocs/pull/5481
+- description: Added the [Manage disk space topic](https://devdocs.magento.com/guides/v2.3/cloud/project/manage-disk-space.html)
+    to the _Cloud Guide_.
+  versions: 2.x
+  type: New topic
+  date: October 10, 2019
+  link: https://github.com/magento/devdocs/pull/5641
+- description: Updated the _Command-line tools reference_:<br/>- Regenerated [bin/magento
+    (Open Source)](https://devdocs.magento.com/guides/v2.3/reference/cli/magento.html)<br/>-
+    Added [bin/magento (Commerce)](https://devdocs.magento.com/guides/v2.3/reference/cli/magento-commerce.html)<br/>-
+    Added [magento-cloud]( https://devdocs.magento.com/guides/v2.3/reference/cli/magento-cloud.html)
+  versions: 2.2.10, 2.3.3
+  type: New topic
+  date: October 10, 2019
+  link: https://github.com/magento/devdocs/pull/5642
+- description: Amazon Sales Channel version 3.0.0 is available in the [Marketplace](https://marketplace.magento.com/magento-module-amazon.html).
```

Comment:
This doesn't seem like an action statement? Like "Added release notes for the Amazon Sales Channel version 3.0.0, which is now available in the Marketplace" or similar.